plugins/Pockets/library/class.pocket.php

```diff
@@ -53,9 +56,12 @@ class Pocket {
    /** $var array The repeat frequency. */
    public $MobileNever = FALSE;
 
+   /** $var boolean Is the pocket an ad? */
+   public $IsAd = FALSE;
```

Comment:
The property should be the type and there should be an `IsAd()` convenience method.